sql-db/hibernate-fulltext-search/src/test/java/io/quarkus/ts/hibernate/search/OpenShiftMysqlMultitenantHibernateSearchIT.java

```diff
@@ -12,19 +11,20 @@
 
 @OpenShiftScenario
 @EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")
-@Disabled("https://github.com/quarkus-qe/quarkus-test-framework/issues/427")
 public class OpenShiftMysqlMultitenantHibernateSearchIT extends AbstractMultitenantHibernateSearchIT {
     static final int ELASTIC_PORT = 9200;
     static final int MYSQL_PORT = 3306;
 
-    @Container(image = "${mysql.80.image}", port = MYSQL_PORT, expectedLog = "Only MySQL server logs after this point", command = " --max-allowed-packet=5526600")
-    static MySqlService base = new MySqlService().withDatabase("base");
+    @Container(image = "${mysql.80.image}", port = MYSQL_PORT, expectedLog = "Only MySQL server logs after this point")
+    static MySqlService base = new MySqlService().withDatabase("base").withProperty("MYSQL_MAX_ALLOWED_PACKET", "5526600");
```

Comment:
Nitpick: can we have this number as a constant? And maybe break this line along the dots?